<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QLineEdit" name="display">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>681</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearAllButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackSpaceButton">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>81</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>BackSpace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SqrtButton">
   <property name="geometry">
    <rect>
     <x>631</x>
     <y>135</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Sqrt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pingfangButton">
   <property name="geometry">
    <rect>
     <x>631</x>
     <y>172</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>x²</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="daoshuButton">
   <property name="geometry">
    <rect>
     <x>631</x>
     <y>209</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>1/x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equButton">
   <property name="geometry">
    <rect>
     <x>631</x>
     <y>246</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="divButton">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>135</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>÷</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mulButton">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>172</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decButton">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>209</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>246</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button9">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>135</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button8">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>135</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button7">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>135</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button6">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>172</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button5">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>172</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button4">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>172</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button3">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>209</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button2">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>209</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button1">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>209</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doubleButton">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>246</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>±</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dotButton">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>246</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button0">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>246</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MCButton">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>135</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>MC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MRButton">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>172</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>MR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MSButton">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>209</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>MS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MPlusButton">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>246</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>M+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="jiechengButton">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>135</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>n!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fanggenButton">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>172</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>yrx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lifanggenButton">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>209</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>3rx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zhishuButton">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>246</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>10x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pingfangButton_2">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>135</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>x²</string>
   </property>
  </widget>
  <widget class="QPushButton" name="xyButton">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>172</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>xy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lifangButton">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>209</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>x3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="duishuButton">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>246</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>lg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sinButton">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>135</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cosButton">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>172</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tanButton">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>209</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>tan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modButton">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>246</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>mod</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ume UI Gothic O5</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>科学计算器</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
